pq/client-pq-tls13.c

```diff
@@ -0,0 +1,266 @@
+/* client-pq-tls13.c
+ *
+ * Copyright (C) 2021 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL. (formerly known as CyaSSL)
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfSSL is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA
+ */
+
+/* the usual suspects */
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+
+/* socket includes */
+#include <sys/socket.h>
+#include <arpa/inet.h>
+#include <netinet/in.h>
+#include <unistd.h>
+
+/* wolfSSL */
+#include <wolfssl/options.h>
+#include <wolfssl/ssl.h>
+#include <wolfssl/wolfio.h>
+#include <wolfssl/wolfcrypt/error-crypt.h>
+
+#define DEFAULT_PORT 11111
+
+#define CERT_FILE "../certs/falcon_level5_root_cert.pem"
+
+#if defined(WOLFSSL_TLS13) && defined(HAVE_SECRET_CALLBACK)
+
+#ifndef WOLFSSL_SSLKEYLOGFILE_OUTPUT
+    #define WOLFSSL_SSLKEYLOGFILE_OUTPUT "sslkeylog.log"
+#endif
+
+/* Callback function for TLS v1.3 secrets for use with Wireshark */
+static int Tls13SecretCallback(WOLFSSL* ssl, int id, const unsigned char* secret,
```

Comment:
This needs wrapped in `HAVE_SECRET_CALLBACK`

```
client-pq-tls13.c:50:12: warning: unused function 'Tls13SecretCallback' [-Wunused-function]
static int Tls13SecretCallback(WOLFSSL* ssl, int id, const unsigned char* secret,
           ^
1 warning generated.
```

Reply:
strange....it is wrapped in it on line 43
`#if defined(WOLFSSL_TLS13) && defined(HAVE_SECRET_CALLBACK)`

Am I missing something?  Can you let me know what flags you configured wolfssl with?